man/systemd.unit.xml

```diff
@@ -1537,24 +1537,29 @@
         </thead>
         <tbody>
           <row>
-            <entry><literal>%n</literal></entry>
-            <entry>Full unit name</entry>
-            <entry></entry>
+            <entry><literal>%b</literal></entry>
+            <entry>Boot ID</entry>
+            <entry>The boot ID of the running system, formatted as string. See <citerefentry><refentrytitle>random</refentrytitle><manvolnum>4</manvolnum></citerefentry> for more information.</entry>
```

Comment:
As long as the lower- and upper-case variants are next to each other, I'm OK with the change.